arcgis-ios-sdk-samples/Maps/Generate offline map (overrides)/OfflineMapParameterOverridesViewController.swift

```diff
@@ -0,0 +1,274 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import ArcGIS
+
+class OfflineMapParameterOverridesViewController: UITableViewController {
+    
+    var parameterOverrides: AGSGenerateOfflineMapParameterOverrides?
+    var map: AGSMap?
+    
+    //MARK: - Outlets
+
+    /// The min scale level for the output. Note that lower values are zoomed further out,
+    /// i.e. 0 has the least detail, but one tile covers the entire Earth.
+    @IBOutlet weak var minScaleLevelSlider: UISlider!
+    
+    /// The max scale level for the output. Note that higher values are zoomed further in,
+    /// i.e. 23 has the most detail, but each tile covers a tiny area.
+    @IBOutlet weak var maxScaleLevelSlider: UISlider!
+    
+    /// The extra padding added to the extent rect to fetch a larger area, in meters.
+    @IBOutlet weak var basemapExtentBufferSlider: UISlider!
+    
+    /// Switch indicating if the system valves layer should be included in the download.
+    @IBOutlet weak var includeSystemValvesSwitch: UISwitch!
+    ///Switch indicating if the service connections layer should be included in the download.
+    @IBOutlet weak var includeServiceConnectionsSwitch: UISwitch!
+    
+    /// The minimum flow rate by which to filter features in the Hydrants layer, in gallons per minute.
+    @IBOutlet weak var minHydrantFlowRateSlider: UISlider!
+    
+    ///Switch indicating if the pipe layers should be restricted to the extent frame.
+    @IBOutlet weak var cropWaterPipesToExtentSwitch: UISwitch!
+    
+    @IBOutlet weak var minScaleLevelLabel: UILabel!
+    @IBOutlet weak var maxScaleLevelLabel: UILabel!
+    @IBOutlet weak var extentBufferLabel: UILabel!
+    @IBOutlet weak var minHydrantFlowRateLabel: UILabel!
+    
+    //MARK: - Actions
+    
+    @IBAction func sliderChangeAction(_ sender: UISlider) {
+        if sender == minScaleLevelSlider {
+            // Disallow a min value greater than the maximum
+            if sender.value > maxScaleLevelSlider.value {
+                sender.value = maxScaleLevelSlider.value
+            }
+        }
+        else if sender == maxScaleLevelSlider {
+            // Disallow a max value less than the minimum
+            if sender.value < minScaleLevelSlider.value {
+                sender.value = minScaleLevelSlider.value
+            }
+        }
+        // Update this slider's text field to display the new value
+        updateTextField(for: sender)
```

Comment:
Since you already know which slider you've gat (tested in the `if…else` statement), you might consider just updating the corresponding label's text inside the `if` and `else if` clauses to prevent testing the slider again.